masterlist.yaml

```diff
@@ -4916,6 +4916,20 @@ plugins:
         itm: 1
         udr: 0
         nav: 0
+  - name: 'RobCo Certified.esm'
+    req: [ *NVSE ]
+    url: [ 'https://www.nexusmods.com/newvegas/mods/43331' ]
+    tag:
+      - Factions
+    dirty:
+      - <<: *quickClean
```

Comment:
Same as for #58, the `<<: *quickClean` just gets overridden by the `util` and `info` values you've specified below.

Reply:
Fixed.